<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsDwgImportBase</class>
 <widget class="QDialog" name="QgsDwgImportBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>228</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DWG/DXF Import</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QgsMessageBar" name="bar" native="true"/>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Import Drawing into GeoPackage</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>CRS</string>
        </property>
        <property name="buddy">
         <cstring>mCrsSelector</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Target package</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_1">
        <property name="text">
         <string>Source drawing</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QgsFileWidget" name="mDrawingFileWidget">
          <property name="dialogTitle">
           <string>Select DWG/DXF file</string>
          </property>
          <property name="filter">
           <string notr="true">*.dwg *.DWG *.dxf *.DXF</string>
          </property>
          <property name="options">
           <set>QFileDialog::ReadOnly</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QCheckBox" name="cbExpandInserts">
          <property name="text">
           <string>Expand block references</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbUseCurves">
          <property name="text">
           <string>Use curves</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbCadStyle">
          <property name="text">
           <string>Use CAD symbology</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="pdImportDrawing">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Import</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QgsProjectionSelectionWidget" name="mCrsSelector">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QgsFileWidget" name="mDatabaseFileWidget">
          <property name="dialogTitle">
           <string>Select GeoPackage Database</string>
          </property>
          <property name="filter">
           <string notr="true">*.gpkg;;*.GPKG</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="lblMessage">
        <property name="text">
         <string notr="true">TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
  </customwidget>
  <customwidget>
   <class>QgsProjectionSelectionWidget</class>
   <extends>QWidget</extends>
   <header>qgsprojectionselectionwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsMessageBar</class>
   <extends>QWidget</extends>
   <header>qgsmessagebar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QgsDwgImportBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QgsDwgImportBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
